Classes/Routing/Router.swift

```diff
@@ -492,6 +492,16 @@ final class Router:
                     )
                 )
             }
+        case .qrScanner:
+            let session = self.appConfiguration.session
```

Comment:
`session` is used only in if-let statement. You can combine these two lines together.